doc/user_manual/libraries.tex

```diff
@@ -0,0 +1,26 @@
+\begin{itemize}
```

Comment:
Do we need to add this to the repo? I think we will automatically generate this file

Reply:
I have added it by mistake